exercises/supervised_stack.livemd

```diff
@@ -30,7 +30,7 @@ defmodule Stack do
 
   def start_link(state, opts) do
     server_name = Keyword.get(opts, :name, __MODULE__)
```

Comment:
I changed this back to keyword, we know that the opts will be a keyword list so it seems clearer to use Keyword than Access. What do you think?